modules/@apostrophecms/ui/ui/apos/components/AposSlat.vue

```diff
@@ -18,6 +18,11 @@
     >
       <div class="apos-slat__main">
         <drag-icon class="apos-slat__control apos-slat__control--drag" :size="13" />
+        <AposContextMenu
+          :button="more.button"
+          :menu="more.menu"
+          :item-props="item"
```

Comment:
rather than pass this down only to emit it from two levels deep, the context menu click event could simply have the action name, which this component should already know. Then this component could emit what's necessary up to `AposInputRelationship.vue`

Reply:
I had the choice between this solution and using the bus to avoid passing events to all the chain of components. I implemented your solution, it might be simpler to understand I guess.